package.json

```diff
@@ -53,14 +75,20 @@
     "react-copy-to-clipboard": "^5.0.4",
     "react-dom": "17.0.2",
     "react-hook-form": "^7.27.0",
+    "react-i18next": "^11.11.4",
     "react-router-dom": "^6.3.0",
     "react-syntax-highlighter": "^15.4.5",
     "react-use-websocket": "^3.0.0",
     "react-virtualized-auto-sizer": "^1.0.6",
+    "recharts": "^2.1.2",
     "regenerator-runtime": "0.13.7",
+    "sass": "^1.49.11",
     "sha3": "^2.1.4",
+    "subscriptions-transport-ws": "^0.9.19",
     "tailwindcss": "^3.0.23",
     "tslib": "^2.0.0",
+    "typescript": "^4.1.2",
+    "use-react-router-breadcrumbs": "^2.0.2",
```

Comment:
chore (blocking): pretty sure we are not using these if my earlier comment about the bread crumb remove is a thing

Reply:
Removed